### PR TITLE
Unshadow audit_rules_unsuccessful_file_modification

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Drop the check from rhel6
- Update the check in shared to account for different AUID
- Update extended-definition's template to apply to also rhel6

#### Rationale:

- Both shared and rhel6 are checking the same thing but from different points of view.
  - share check iterates over each rule syscal, and
  - rhel6 check iterates over each error (EACCES/EPERM)
- I've favored the shared check, as its regex'es are better in general, supports augenrules, and is more readable.

### CODE REVIEW

File: shared/checks/oval/audit_rules_auditctl.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <metadata>
       <title>Record Any Attempts to Run semanage</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_rhel</platform>
```
Comment (mpreisler) [inline shared/checks/oval/audit_rules_auditctl.xml]: how could this OVAL possibly work on RHEL6? It relies on systemd!!

**Comment by cipherboy** [comment]
@openscap-ci test this please.

File: shared/checks/oval/audit_rules_auditctl.xml (hunk 1 above)
Comment (yuumasato) [reply shared/checks/oval/audit_rules_auditctl.xml]: Well, that was a huge oversight, :D

Comment (yuumasato) [reply shared/checks/oval/audit_rules_auditctl.xml]: PR #3056 should address this issue.

Comment (mpreisler) [reply shared/checks/oval/audit_rules_auditctl.xml]: Rebase this please

Comment (yuumasato) [reply shared/checks/oval/audit_rules_auditctl.xml]: Rebased.

**Pull request merged**